src/middlewared/middlewared/api/v25_10_0/system_advanced.py

```diff
@@ -21,45 +21,69 @@
 
 class SystemAdvancedEntry(BaseModel):
     id: int
+    """Unique identifier for the system advanced configuration."""
     advancedmode: bool
+    """Enable advanced mode to show additional configuration options in the web interface."""
     autotune: bool
     """Execute autotune script which attempts to optimize the system based on the installed hardware."""
     kdump_enabled: bool
+    """Enable kernel crash dumps for debugging system crashes."""
     boot_scrub: PositiveInt
+    """Number of days between automatic boot pool scrubs."""
     consolemenu: bool
     """Enable console menu. Default to standard login in the console if disabled."""
     consolemsg: bool
     """Deprecated: Please use `consolemsg` attribute in the `system.general` plugin instead."""
     debugkernel: bool
+    """Enable debug kernel for additional logging and debugging capabilities."""
     fqdn_syslog: bool
+    """Include the full domain name in syslog messages."""
     motd: str
+    """Message of the day displayed after login."""
     login_banner: str = Field(max_length=4096)
+    """Banner message displayed before login prompt."""
     powerdaemon: bool
+    """Enable the power management daemon for automatic power management."""
     serialconsole: bool
+    """Enable serial console access."""
     serialport: str
+    """Serial port device for console access."""
     anonstats_token: str
+    """Token used for anonymous statistics reporting."""
     serialspeed: Literal['9600', '19200', '38400', '57600', '115200']
+    """Baud rate for serial console communication."""
     overprovision: NonNegativeInt | None
+    """Percentage of SSD overprovisioning to reserve for wear leveling."""
     traceback: bool
+    """Enable generation and saving of tracebacks for debugging."""
     uploadcrash: bool
+    """Automatically upload crash reports to iXsystems for analysis."""
     anonstats: bool
+    """Enable anonymous usage statistics reporting to help improve TrueNAS."""
     sed_user: Literal['USER', 'MASTER']
+    """SED (Self-Encrypting Drive) user type for drive encryption."""
     sysloglevel: Literal['F_EMERG', 'F_ALERT', 'F_CRIT', 'F_ERR', 'F_WARNING', 'F_NOTICE', 'F_INFO', 'F_DEBUG']
+    """Minimum log level for syslog messages. F_EMERG is most critical, F_DEBUG is least critical."""
     syslogserver: str = NotRequired
     """When defined, logs of `sysloglevel` or higher are sent."""
```

Comment:
The associated tooltip provides the proper documentation.
```suggestion
    """Remote syslog server DNS hostname or IP address. Nonstandard port numbers can be used by adding a colon and the port number to the hostname, like mysyslogserver:1928."""
```